GitCommands/CommitMessageManager.cs

```diff
@@ -42,6 +44,10 @@ public sealed class CommitMessageManager : ICommitMessageManager
         private Encoding _commitEncoding;
         private IFileSystem _fileSystem;
 
+        private const string _cannotReadCommitMessage = "Cannot read commit message";
```

Comment:
TranslationString cannot be used in GitCommands

Reply:
They can be moved to `Strings.cs`.

Reply:
Which file?

Reply:
Open the command window (menu `View` | `Other Windows` | `Command Window`) and type "open str" ...

Reply:
Of course I found a file called Strings.cs, but that is still in GitUI

Reply:
You're right. But we have another: `ResourceManager\Strings.cs`.